src/features/Account/ManageAccountContainer.tsx

```diff
@@ -80,7 +87,7 @@ class ManageAccountContainer extends React.Component<
         password: getNestedAttr(props, ['location', 'state', 'password']) || '',
         phoneNumber: '',
         pronoun: '',
-        shirtSize: '',
+        gender: '',
```

Comment:
Consider making the shirtSize / gender swap a different PR. It makes reviews much easier!